truss/remote/baseten/custom_types.py

```diff
@@ -13,12 +15,28 @@ class DeployedChainlet(pydantic.BaseModel):
     oracle_name: str
 
 
-class ChainletData(pydantic.BaseModel):
-    name: str
-    oracle_version_id: str
+class ChainletArtifact(pydantic.BaseModel):
+    truss_dir: pathlib.Path
     is_entrypoint: bool
+    display_name: str
+    name: str
```

Comment:
What's the difference here? Are both needed?

Reply:
Unfortunately yes... `display_name` is used for the mutation and `name` is used for the dynamic config.